fastlane/Fastfile

```diff
@@ -22,6 +22,26 @@ platform :ios do
     sh 'git config --global --unset url."https://github.com/".insteadOf'
   end
 
+  desc "Commit version bump in _versions"
```

Comment:
missing indentation.
Also the description is not accurate. It's not just about `_versions`